skiko/src/jvmMain/cpp/common/shaper/interop.cc

```diff
@@ -46,7 +46,7 @@ namespace skija {
 
             void onLoad(JNIEnv* env) {
                 jclass local = env->FindClass("org/jetbrains/skia/shaper/HbIcuScriptRunIterator");
-                cls  = static_cast<jclass>(env->NewGlobalRef(local));            
+                cls  = static_cast<jclass>(env->NewGlobalRef(local));
```

Comment:
Generally, all cached classes and fields shall go away after we done with this conversion.

Reply:
The line was changed not intentionally, but by IDE formatting.  

as for now, it's still needed as it's used in one of the Shaper jni implementations. So it might be removed later.